### ISSUE TITLE
Wyzesense integration malfunctions after enabling HACS debugging in configuration.yaml logger

### ISSUE THREAD

**Issue by carver7** [kevinvincent/ha-wyzesense#62]
First, thanks for making this integration! It has definitely opened up a whole lot of options for me with home data collection, automation, and pushing my Wyze senors to Homekit and otherwise freeing them from a closed cloud-based system.

This isn't a big deal to me personally, but I'm bringing this apparent bug up in case it's causing anyone difficulties:
 
Wyzesense integration malfunctions after enabling HACS debugging in configuration.yaml logger:
e.g.
configuration.yaml :
```
logger:
  default: info
  logs:
    hacs: debug  #<-- Causes wyzesense component improper loading and other wyzesense error msgs in home-assistant.log
```

(My setup: hassio alt generic linux install in Docker on RPi 4; Home Assistant v 0.101.3, HACS 0.16.3)

**Comment by kevinvincent** [comment]
Huh interesting, by improper loading and other wyzesense errors can you clarify what you're seeing? A paste from the log file would be very much appreciated!

**Comment by carver7** [comment]
@kevinvincent 
I haven't been able to reproduce this issue over the last few upgrades. I just tested it again with HA v0.103.6, and no problems. 
I'll go ahead and close the issue since I'm not seeing it being experienced other users as well.
 
(Unfortunately, I didn't save the log from the last time I experienced the errors, but if I happen to experience it again, I'll be sure to provide it to you)

Thumbs up and thanks for sharing!
